UndertaleModLib/Models/UndertaleString.cs

```diff
@@ -62,6 +62,7 @@ public void Dispose()
     /// <returns>A string that represents the current object.</returns>
     public string ToString(bool isGMS2)
     {
+        // TODO: someone clean this up please. this seems insane for a tostring method.
```

Comment:
I think this occurs with corrupted or partially-loaded VM code, which otherwise could crash any code calling this... i.e., decompiling code and the code editors. It should probably stay here.

Reply:
Well I mentioned "clean this up". not "outright remove".
There's a few things for example at a first glance, where one should use a stringbuilder instead of just concatonating over and over to an already existing string. Or using Trim methods instead.
To my knowledge, this method is called semi-frequently, so it should really get a second pass + some documentation. 

Although now i'm curious how "corrupted or partially loaded VM code" can even happen...